mindsdb/integrations/handlers/langchain_handler/skill_tool.py

```diff
@@ -19,20 +21,25 @@ def _make_text_to_sql_tools(skill: db.Skills, llm, executor) -> List:
         raise ImportError('To use the text-to-SQL skill, please install langchain with `pip install langchain`')
     database = skill.params['database']
     tables = skill.params['tables']
-    tables_to_include = [f'{database}.{table}' for table in tables]
+    # tables_to_include = [f'{database}.{table}' for table in tables]
```

Comment:
Leftover? If so, remove.

Reply:
done.
please take attention, this function will be removed in https://github.com/mindsdb/mindsdb/pull/8792